CONTRIBUTING.md

```diff
@@ -107,8 +103,12 @@ following rules before you submit a pull request:
    functionality is useful in practice and, if possible, compare it
    to other methods available in scikit-learn.
 
--  Documentation and high-coverage tests are necessary for enhancements
-   to be accepted.
+-  Documentation and high-coverage tests are necessary for enhancements to be
+   accepted. Bug-fixes or new features should be provided with non-regression
+   tests. Tests that verify the correct behavior of the fix or feature. In this
```

Comment:
I would be even more explicit for bug-fix to say that the test needs to be failing on master and passing in the PR branch.